specification/cat/indices/CatIndicesRequest.ts

```diff
@@ -36,5 +37,6 @@ export interface Request extends CatRequestBase {
     health?: HealthStatus
     include_unloaded_segments?: boolean
     pri?: boolean
+    time?: Duration
```

Comment:
This should be `TimeUnit`, as the [docs](https://www.elastic.co/guide/en/elasticsearch/reference/current/cat-indices.html) says "Unit used to display time values" and links to [time unit definitions](https://www.elastic.co/guide/en/elasticsearch/reference/current/api-conventions.html#time-units).